resources/js/Organisms/Form.tsx

```diff
@@ -1,19 +1,21 @@
 import React, { useState, useEffect, useRef } from "react";
 import { Inertia } from "@inertiajs/inertia";
+import { usePage } from "@inertiajs/inertia-react";
 
 interface IFormProps<IValues extends Record<string, any>> {
     /* The http path that the form will be posted to */
     action: string;
     initialValues: IValues;
     button: string;
-    validate: (values: IValues) => IErrors<IValues>;
+    validate?: (values: IValues) => IErrors<IValues>;
     /* A prop which allows content to be injected */
     render: (
         val: IValues,
         handleChange: (
             e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>,
         ) => void,
         errors: IErrors<IValues>,
+        errorsBE: IErrors<IValues>,
```

Comment:
this name isn't that clear, maybe `errorsFromServer` or `errorsFromBackend` or something like that would be a bit easier for newcomers to understand?